README.md

```diff
@@ -213,6 +213,7 @@ Options:
   -j, [--skip-js-linters], [--no-skip-js-linters]      # Skip installing JavaScript linting files
   -L, [--ruby-linters], [--no-ruby-linters]            # Install ruby linting files, tasks, and configs
   -H, [--heroku-deployment], [--no-heroku-deployment]  # Install files necessary for deploying to Heroku
+  -B, [--bootstrap], [--no-bootstrap]  # Install files necessary for bootstrap support.
```

Comment:
@justin808 @yorzi to be consistent with what we've got so far, if the default is that the bootstrap code is generated, and the user would need to specifically pass an option to turn it off, then we should use the `--skip-bootstrap` option not just `bootstrap`, just like how we're using `--skip-js-linters` since by default they are on